nautobot/extras/tests/test_api.py

```diff
@@ -3464,6 +3544,48 @@ def test_update_tag_content_type_unchanged(self):
         self.assertEqual(list(tag.content_types.all()), tag_content_types)
 
 
+#
+# Team
+#
+
+
+class TeamTest(APIViewTestCases.APIViewTestCase):
+    model = Team
+    create_data = [
+        {
+            "name": "Team 1",
+            "phone": "12345212",
+            "email": "team1@gmail.com",
```

Comment:
Probably best to use "definitively" fake phone numbers (555-01xx) and emails (@example.com) here and in the other tests.